packages/apollo-server/src/index.ts

```diff
@@ -31,6 +36,28 @@ export class ApolloServer<Context> extends ExpressServer {
 
     opts.app = express();
     super(opts);
+
+    if (!opts.disableHealthCheck) {
+      //uses same path as engine
+      opts.app.use('/.well-known/apollo/server-health', (req, res, next) => {
+        //Response follows https://tools.ietf.org/html/draft-inadarei-api-health-check-01
+        res.type('application/health+json');
+
+        if (opts.onHealthCheck) {
+          opts
+            .onHealthCheck(req)
```

Comment:
Do we need to guard against `onHealthCheck` being a `then`-able?

Reply:
Thoughts on providing an API that doesn't _require_ a `Promise` to be returned but still welcomed a `Promise`)?  In other words, it could be a normal `function` or an `async function` or a `Promise`?

Reply:
Never mind.  Let's re-consider this second point later.  This isn't going to be highly trafficked right now.